scss/components/organisms/accordions/index.scss

```diff
@@ -0,0 +1,9 @@
+@charset 'UTF-8';
+
+@import
+  "accordion-button-styles",
```

Comment:
This component had a lot of mixins in it. Are these all really to be re-used in other components?